assemblies/chap-optaplanner-vaccination.adoc

```diff
@@ -0,0 +1,251 @@
+
+image::optimizer/vaccinationSchedulingValueProposal.png[]
+
+
+[id="con-continuous-planning_{context}"]
+== Continuous planning
+
+Continuous planning is the technique of managing one or more upcoming planning periods at the same time and repeating that process monthly, weekly, daily, hourly, or even more frequently. The planning window advances incrementally by a specified interval. The following illustration shows a two week planning window that is updated daily:
+
+image::optimizer/vaccinationSchedulingContinuousPlanning.png[]
+
+The two week planning window is divided in half. The first week is in the published state and the second week is in the draft state.  People are assigned to appointments in both the published and draft parts of the planning window. However, only people in the published part of the planning window are notified of their appointments. The other appointments can still change easily in the next run. Doing this gives OptaPlanner the flexibility to change the appointments in the draft part when you run the solver again, if necessary. For example, if a person who needs a second dose has a ready date of Monday and an ideal date of Wednesday, OptaPlanner does not have to give them an appointment for Monday if you can prove OptaPlanner can demonstrate that it can give them a draft appointment later in the week.
+
+You can determine the size of the planning window but just be aware of the size of the problem space. The problem space is all of the various elements that go into creating the schedule. The more days you plan ahead, the larger the problem space.
+
+
+[id="con-pinned-planning-entities_{context}"]
+== Pinned planning entities
+
+If you are continuously planning on a daily basis, there will be appointments within the two week period that are already allocated to people. To ensure that appointments are not double-booked, OptaPlanner marks existing appointments as allocated by pinning them. Pinning is used to anchor one or more specific assignments and force OptaPlanner to schedule around those fixed assignments.  A pinned planning entity, such as an appointment, does not change during solving.
+
+Whether an entity is pinned or not is determined by the appointment state. An appointment can have five states : `Open`, `Invited`, `Accepted`, `Rejected`, or `Rescheduled`.
+
+NOTE: You do not actually see these states directly in the quick start demo code because the OptaPlanner engine is only interested in whether the appointment is pinned or not.
```

Comment:
```suggestion
NOTE: You do not see these states directly in the quick start demo code because the OptaPlanner engine is only interested in whether the appointment is pinned or not.
```